web/source/kmwcallback.js

```diff
@@ -54,7 +54,7 @@
         Lelem._KeymanWebSelectionEnd=Le;
         keymanweb._IgnoreNextSelChange = 0;
         if(Ptext!=null)keymanweb.KO(0, Lelem, Ptext);
-        if(typeof(PdeadKey)!='undefined') keymanweb.KDO(0, Lelem, PdeadKey);
+        if((typeof(PdeadKey)!=='undefined') && (PdeadKey !== null)) keymanweb.KDO(0, Lelem, PdeadKey);
```

Comment:
Looking at various keyboards in the `s.keyman.com` repo, `KeymanWeb.KT()` is called in a variety of ways:
````
    KeymanWeb.KT('');  // tibetan_ewts_to_unicode-1.1.js
    KeymanWeb.KT(null);  // korean_rr-1.1.js
    KeymanWeb.KT('');  // japanese-1.0.js
    KeymanWeb.KT('');  // european2-1.6.js
    KeymanWeb.KT(attributes['x'].value,0); // ditto
    KeymanWeb.KT(innerHTML); // ditto
    KeymanWeb.KT('') // chinese-1.0.js
````
Can we check that each of these usages will be acceptable according to the `KT` declaration and comment? Then, I think we can go ahead with this.